### PR TITLE
Update spark.md

### LINKED ISSUE #972: Document how to debug/profile spark hellbender tools

### PR DESCRIPTION
**Pull request by akiezun**
fixes #972 (only running, not debugging)

### CODE REVIEW

**Comment by akiezun** [comment]
@tomwhite can you review?

File: spark.md [hunk 1]
```diff
@@ -1,29 +1,15 @@
 Hellbender on Spark
```
Comment (tomwhite) [inline spark.md]: s/Hellbender/GATK/g

File: spark.md [hunk 2]
```diff
@@ -1,29 +1,15 @@
...
-Running Hellbender Commands Locally on Spark Dataflow (Without a Cluster)
-------------
-
-This allows you to run Hellbender Dataflow command lines using Spark running in the local JVM. It is useful for testing
-commands against local files using the Spark Dataflow runner.
+This allows you to run GATK4 command lines using Spark running in the local JVM. It is useful for testing
+commands against local files using the Spark runner.
 
 First, [download](https://spark.apache.org/downloads.html) and unpack the Spark tarball on your local machine.
 Version 1.3.1 for Hadoop 2.6 is recommended.
```
Comment (tomwhite) [inline spark.md]: This should be 1.5.x now.

**Comment by tomwhite** [comment]
A couple of minor nits but otherwise +1.

**Pull request merged**